### ISSUE TITLE
Investigating raising the maximum brush count

### ISSUE THREAD

**Issue by solarisstar** [solarisstar/rpgxEF#83]
Right now it's 2**16, even a doubling would allow for much more complicated maps.

**Comment by MJLHThomassen** [comment]
Yes, raising it SHOULD be trivial (As long as users have enough RAM i suppose), however i have no idea how this will impact server-client interactions (if at all). It hink the 2^16 was chosen because its stored in shorts? (just a guess). So raising it might not be as trivial as it seems. But well see and definately should adjust this.

Also, we should see if its possible to increase **the Brush-Entity count** (the amount of brush-entities that can exist like doors etc). Thats the real bottleneck for more complicated maps and might be more work since it involves entities and definately involves server-client interaction.

**Comment by solarisstar** [comment]
(resurrecting this topic)

The entity count is even more limited at 1024. Changing the defined constants is the easy part; like you said, refactoring the net code to support bigger maps will be much harder. I would imagine we'll also start hitting performance problems since the entire game is single threaded.
